src/router.js

```diff
@@ -16,14 +16,23 @@ export const routeGuard = (store, apollo) => async (to, from, next) => {
     next(path)
     return
   }
-
   if (
     to.meta.feature &&
     !(store.state.connection.network === "testnet") && // TODO remove once we have Hasura integrated in e2e tests
-    !(await featureAvailable(apollo, store.state.connection.network, to))
+    !(await featureAvailable(apollo, store.state.connection.network, to)) &&
+    !(
+      (await featureAvailable(apollo, store.state.connection.network, to)) ===
+      null
+    )
   ) {
     next(`/feature-not-available/${to.meta.feature}`)
```

Comment:
we should embed these and not reroute. that was a bad decision of me before. on a refresh you are stuck there

Reply:
but later